packages/arb-token-bridge-ui/src/hooks/useTransactionHistory.ts

```diff
@@ -221,48 +244,64 @@ const useTransactionHistoryWithoutStatuses = (
     cctpTransfersTestnet.isLoadingDeposits ||
     cctpTransfersTestnet.isLoadingWithdrawals
 
+  const fetcher = useCallback(
```

Comment:
Extracted deposits and withdrawals fetching to a single method because they start to share more logic